ios/Classes/FlutterBleLibPlugin.m

```diff
@@ -167,6 +169,16 @@ - (void)isDeviceConnected:(FlutterMethodCall *)call result:(FlutterResult)result
 
 }
 
+- (void)observeConnectionState:(FlutterMethodCall *)call result:(FlutterResult)result {
+    BOOL emitCurrentValue = call.arguments[ARGUMENT_KEY_EMIT_CURRENT_VALUE];
+    if (emitCurrentValue == YES) {
+        [self isDeviceConnected:call result:result];
```

Comment:
The current value should be emitted to the stream, not returned in the result.

Reply:
Pushed changes to address this issue.